### DISCUSSION TITLE
Why ı can change only one head model

### DISCUSSION THREAD

**Discussion by Magiciandseq3** [nonamecrackers2/crackers-wither-storm-mod#1962] [Q&A] [answered]
Hello Crackers ı can only change one head model. Can ı change  all wither storm heads  too?

**Comment by BluePineappleGuy** [comment]
So only 1 of the heads were changed? Maybe they have different models, or it might be a bug.

**Reply by Magiciandseq3** [reply]
Yes:(

**Comment by nonamecrackers2** [comment, marked as answer]
You should be able to, they're all under different model groups so they should be separate. I'd recommend asking in our [Discord](https://discord.gg/cracker-s-modded-community-987817685293355028) in the coding channel for this question, since there are people who've modified the models that may be able to help you.
